administrator/language/en-GB/plg_twofactorauth_email.sys.ini

```
@@ -0,0 +1,7 @@
+; Joomla! Project
+; (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_TWOFACTORAUTH_EMAIL="Two Factor Authentication - Authentication Code by Email"
+PLG_TWOFACTORAUTH_EMAIL_XML_DESCRIPTION="<p>Use time-limited, six-digit security codes sent to you by email.</p>"
```

```suggestion
PLG_TWOFACTORAUTH_EMAIL_XML_DESCRIPTION="Use time-limited, six-digit security codes sent to you by email."
```

The html should not be needed